package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --configuration development",
     "start-proxy": "ng serve --proxy-config proxy.conf.json",
     "build": "ng build",
-    "build-prod": "ng build --prod --aot --build-optimizer --outputPath=./dist/out",
```

Comment:
see above, for production (drone builds) this shouldn't be changed, right?

Reply:
yes, that is correct, I reverted those changes